setarch.c

```diff
@@ -22,16 +22,15 @@ struct exec_domain {
 void setarch(const char *arch)
 {
 	static struct exec_domain domains[] = {
-		/* Placeholder for host execution domain */
-		{ "", 0 },
+		{ "", 0 },			/* Placeholder for host execution domain */
```

Comment:
micro-nit: align the comments with spaces -- between tabs at 4 columns and tabs at 8 columns, using tabs for aligning things is usually ill-fated

Reply:
Done